infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/VcsSshKeysProvisioner.java

```diff
@@ -210,22 +211,32 @@ private void mountConfigFile(PodSpec podSpec, String sshConfigMapName) {
    *
    * <pre>
    * host github.com
-   * HostName github.com
    * IdentityFile /etc/ssh/github-com/ssh-privatekey
    * </pre>
    *
-   * @param host the host of version control service (e.g. github.com, gitlab.com and etc)
+   * or
+   *
+   * <pre>
+   * host *
+   * IdentityFile /etc/ssh/default-123456/ssh-privatekey
+   * </pre>
+   *
+   * @param name the of key given during generate for vcs service we will consider it as host of
+   *     version control service (e.g. github.com, gitlab.com and etc) if name starts from
+   *     "default-{anyString}" it will be replaced on wildcard "*" host name. Name with format
+   *     "default-{anyString}" will be generated on client side by Theia SSH Plugin, if user don't
```

Comment:
`will be generated on client side by Theia SSH Plugin` -> `is generated on client side by Theia SSH Plugin`

`if user don't` -> `if user doesn't`

Reply:
fixed